plugin/src/main/java/org/openrewrite/gradle/ResolveRewriteDependenciesTask.java

```diff
@@ -73,6 +73,7 @@ public Set<File> getResolvedDependencies() {
                     deps.create("org.openrewrite:rewrite-protobuf:" + rewriteVersion),
                     deps.create("org.openrewrite:rewrite-xml:" + rewriteVersion),
                     deps.create("org.openrewrite:rewrite-yaml:" + rewriteVersion),
+                    deps.create("org.openrewrite.recipe:rewrite-all:" + extension.getRewriteAllVersion()),
```

Comment:
@sambsnyd, speaking of `rewrite-all` depending on all recipe libraries... This by default now kinda means that end users now have all recipes available. Are we intentionally wanting for that to be the case or should we disable transitives here, specifically?